CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Unreleased
 
+* Depricated: [ Resolution#ipfsHash, Resolution#ipfsRedirect, Resolution#email ] -> use Resolution#ipfs || Resolution#whois
```

Comment:
`whois` as an API method can be counter intuitive. Suppose I am developer working with the library for the 1st time. I need to get an email associated with domain. How would realize I need to call `whois` and check the result? I think we need to keep separated methods for getting separated values. Having `whois` at the moment is weird: why would `url` be a part of `whois` and `ipfs` would not? It is like http website is considered a part of my identity while ipfs website is not. Same thing with IPFS. 


This bundle query methods serve no purpose. What would be the reason for me to have a `whois` method? Why would I need all of that together? There is no real task for that.